web/Makefile

```diff
@@ -1,3 +1,16 @@
+
+.PHONY: all
+all:
+	$(MAKE) get-deps
+	$(MAKE) build
+
+# .PHONY: clean
+# .PHONY: fmt
+# .PHONY: check
+# .PHONY: get-deps
+# .PHONY: build
+# .PHONY: test
+
```

Comment:
these were the stages from the previous `Makefile` at `webui/react`. leaving them here as a reminder to test.